### DISCUSSION TITLE
`vc-selection-indicator`选择重叠的多个GeoJSON对象，`pickEvt`中只能拿到一个对象。

### DISCUSSION THREAD

**Discussion by houluy** [zouyaoji/vue-cesium#223] [Q&A]
作者大大，您好，我在使用`vc-selection-indicator`时，如果遇到重叠的GeoJSON对象，发现`pickEvt`的参数`e`只能拿到其中一个对象。如果需要拿到所有选取点位的GeoJSON对象，还是得利用组件方法`getPickedFeatures`里的`value.features`才可以得到所有被选中的对象。

**所以想请问作者大大，是不是我使用方法有误？**

如下是我的部分代码：

```vue
<vc-selection-indicator ref="selectionIndicationRef" @pick-evt="pickEvt">
</vc-selection-indicator>

<script lang="ts" setup>
const pickEvt = (e: any) => {
  // Push an selected property to selectedProperties array
  if (e === undefined || e.cesiumObject === undefined) {//Empty selection
    return
  }

  // Picked features must be extracted from ref object
  console.log(e) 
  console.log(selectionIndicationRef.value.getPickedFeatures().value.features)
  ...
```

我的GeoJSON对象图片：
![pick](https://user-images.githubusercontent.com/11565649/178669097-4ab56890-b5a1-44ec-b212-5ddb07427869.png)

pick之后的打印：
![InkedpickRes](https://user-images.githubusercontent.com/11565649/178669271-aed9b512-eebe-48ea-8023-eebab51f611f.jpg)

**Comment by zouyaoji** [comment]
看一下 pickedFeature 是不是有数组，没有估计就只有用你说的这个办法获取了。
![image](https://user-images.githubusercontent.com/24430116/178735443-532deabd-7a83-4bbe-b570-132444475fbd.png)
